### PR TITLE
support zeros_like and add test cases for remove_ops pass

### PR DESCRIPTION
**Pull request by shivadbhavsar**
Merge #172 first: the simplification added here exposes a bug in some models that is fixed by that PR

This PR:
1. Add tests for the remove_ops pass that is run in the dynamo workflow
2. Add simplification to remove `torch.ops.aten.zeros_like.default` and replace it with a constant tensor instead

### CODE REVIEW

File: py/torch_migraphx/dynamo/passes/remove_ops.py [hunk 1]
```diff
@@ -102,13 +102,18 @@ def _remove_const_like_ops(gm: torch.fx.GraphModule):
...
             torch.ops.aten.full.default: torch.full,
+            torch.ops.aten.zeros_like.default: torch.full,
         }
         for node in gm.graph.nodes:
             if node.op == "call_function" and node.target in const_ops.keys():
                 og_node = node
                 size = og_node.meta['tensor_meta'].shape
                 dtype = og_node.meta['tensor_meta'].dtype
-                value = node.args[1]
+
```
Comment (bpickrel) [inline py/torch_migraphx/dynamo/passes/remove_ops.py:112]:
```suggestion
                value = 0 if node.target == torch.ops.aten.zeros_like.default else node.args[1]
```

File: tests/dynamo/passes/dynamo_passes_test_utils.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import torch
+from torch.fx.passes.shape_prop import ShapeProp
+
+
+def generate_func_gm(func,
+                     inputs=None,
```
Comment (bpickrel) [inline tests/dynamo/passes/dynamo_passes_test_utils.py:6]: I think it's legal to simply default these here, e.g. 
```
constants = {},
etc
```

**Review by bpickrel — APPROVED** [review]
A couple of style comments

File: tests/dynamo/passes/dynamo_passes_test_utils.py (hunk 2 above)
Comment (shivadbhavsar) [reply tests/dynamo/passes/dynamo_passes_test_utils.py:6]: cant do that actually, creating mutable objects as argument defaults can have unintended side effects and is usually avoided. See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for a simple example

**Pull request merged**